# serializer version: 1
# name: TestMatomoSettings.test_url_without_idsite_raises_value_error
  '''
  1 validation error for MatomoSettings
    Value error, MATOMO_IDSITE must be an INT, got: 'None' (<class 'NoneType'>) [type=value_error, input_value={'matomo_url': 'http://example.com'}, input_type=dict]
      For further information visit https://errors.pydantic.dev/2.7/v/value_error
  '''
# ---
